version.json

```diff
@@ -1,4 +1,4 @@
 {
   "name": "mgmorbs/microservices",
-  "version": "0.3.1"
+  "version": "0.4.0"
```

Comment:
did you actually publish a dev version of this orb yet?  we should do this inside of circleCI and be sure to run `circleci config validate` to ensure the config is valid.

Reply:
no, i did not; i wanted to get team opinion before
config validate is automatic on this repo (one of the checks of PR) and also on tag in this repo we will get auto-publishing (take a look on `.circleci/config.yaml`)

Reply:
more just curious, but we should come to a process on this.  I think it's safe to always publish to dev with `npm version` or in Circle, since dev is just arbitrary version tags and production orbs are all SemVer.

Reply:
what I've done:
- on each `branch` push there will be published (after validating of course) orb with tag `@dev:BRANCHNAME`
- on `git tag` there will be published a new "public" version of this orb (with the version from `version.json`)

so you can use a development version of this orb in your `.circleci/config.yaml` e.g.:
```yaml
orbs:
  # Repo: https://github.com/MGMDV-Orbs/microservices
  # Latest Version: https://circleci.com/orbs/registry/orb/mgmorbs/microservices
  ms: mgmorbs/microservices@dev:f/SRV-188
```

this repo now needs one approve from reviewers and to `validate` phase to pass for every PR